### PR TITLE
Listen for the install referrer intent

### PR DESCRIPTION
**Pull request by albertolalama**
* The Play Store fires an install referrer intent when it installs an app from a link that contains the `referrer` parameter. This change listens for that intent and return it to the client when `getUri`is called.
* To test locally (as the real test entails uploading a release to the Play Store):
  * Build and install the app, don't open it.
  * From a terminal run:

    ```
    adb shell am broadcast -a com.android.vending.INSTALL_REFERRER --es "referrer" "ss://YWVzLTI1Ni1jZmI6dXByb3h5MTIzQDE2Mi4yNDMuMTI0LjE3Njo4MDgw#Install%20Referrer"
    ```
  * Manually launch the app. You should see a server "Install Referrer" added.
* No changes in uProxy are necessary, as it already call `getUri` on startup.

### CODE REVIEW

**Comment by albertolalama** [comment]
@jab is finding a maintained fork of this plugin so we can upstream the changes. We were forced to upload the its source when the original repo was taken down. If no good candidate is found, I'll be happy to merge it into uproxy-air.

I will update the README in a follow up PR.

File: src/android/WebIntent.java [hunk 1]
```diff
@@ -214,4 +221,18 @@ void sendBroadcast(String action, Map<String, String> extras) {
...
+    // Receiver that listens for com.android.vending.INSTALL_REFERRER, an intent sent by the
+    // Play Store on installation when the referrer parameter of the install URL is populated:
+    // https://play.google.com/store/apps/details?id=|APP_ID|&referrer=|REFERRER|
+    public static class ReferralReceiver extends BroadcastReceiver {
+        private static final String LOG_TAG = "ReferralReceiver";
+
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            // Store the install referrer so we can return it when getUri is called.
+            installReferrer = intent.getStringExtra("referrer");
```
Comment (jab) [inline src/android/WebIntent.java]: Hey @albertolalama, I just noticed this method modifies the static member `installReferrer` here but the method itself is not static (as the abstract method it overrides isn't). Should the `installReferrer` field be made non-static too?

Comment (jab) [inline src/android/WebIntent.java]: (Oops, I missed that this inner (`ReferralReceiver`) class is itself static, so no need. Sorry about that, please ignore!:)

**Comment by jab** [comment]
@albertolalama @fortuna check it out!:

https://github.com/chrisekelley/cordova-webintent/pull/3/files#diff-04c6e90faac2675aa89e2176d2eec7d8

As soon as https://github.com/chrisekelley/cordova-webintent/pull/4 is merged we should be able to depend on it as our upstream instead of maintaining our own fork. ✨

**Comment by jab** [comment]
chrisekelley/cordova-webintent#4 has now been merged! Just opened chrisekelley/cordova-webintent#6 to get a new release cut that we'll be able to depend on.

**Comment by jab** [comment]
@fortuna et al. please see uProxy/uproxy-air#127 :)

**Pull request merged**